src/compiler/utilities.ts

```diff
@@ -309,6 +309,14 @@ namespace ts {
         return getSourceTextOfNodeFromSourceFile(getSourceFileOfNode(node), node, includeTrivia);
     }
 
+    /**
+     * Gets flags that control emit behavior of a node.
+     */
+    export function getEmitFlags(node: Node): EmitFlags | undefined {

```

Comment:
Why was this moved here? All of the other `emitNode` related functions are still in factory, including the corresponding `setEmitFlags` function.